2020/04/09 15:21:12 [error] 7#7: *1 open() "/var/www/jeecg-boot/sys/randomImage/1586445672615" failed (2: No such file or directory), client: 172.26.0.1, server: localhost, request: "GET /jeecg-boot/sys/randomImage/1586445672615?_t=1586445672 HTTP/1.1", host: "localhost", referrer: "http://localhost/user/login?redirect=%2F"
2020/04/09 15:21:25 [error] 7#7: *1 open() "/var/www/jeecg-boot/sys/randomImage/1586445685332" failed (2: No such file or directory), client: 172.26.0.1, server: localhost, request: "GET /jeecg-boot/sys/randomImage/1586445685332?_t=1586445685 HTTP/1.1", host: "localhost", referrer: "http://localhost/user/login?redirect=%2F"
2020/04/09 15:21:26 [error] 7#7: *1 open() "/var/www/jeecg-boot/sys/randomImage/1586445686441" failed (2: No such file or directory), client: 172.26.0.1, server: localhost, request: "GET /jeecg-boot/sys/randomImage/1586445686441?_t=1586445686 HTTP/1.1", host: "localhost", referrer: "http://localhost/user/login?redirect=%2F"
2020/04/09 15:22:24 [error] 7#7: *6 open() "/var/www/jeecg-boot/sys/randomImage/1586445744551" failed (2: No such file or directory), client: 172.26.0.1, server: localhost, request: "GET /jeecg-boot/sys/randomImage/1586445744551?_t=1586445744 HTTP/1.1", host: "localhost", referrer: "http://localhost/user/login?redirect=%2F"
2020/04/09 15:22:26 [error] 7#7: *6 open() "/var/www/jeecg-boot/sys/randomImage/1586445746108" failed (2: No such file or directory), client: 172.26.0.1, server: localhost, request: "GET /jeecg-boot/sys/randomImage/1586445746108?_t=1586445746 HTTP/1.1", host: "localhost", referrer: "http://localhost/user/login?redirect=%2F"
2020/04/09 15:22:30 [error] 7#7: *6 open() "/var/www/user/login" failed (2: No such file or directory), client: 172.26.0.1, server: localhost, request: "GET /user/login?redirect=%2F HTTP/1.1", host: "localhost"
2020/04/09 15:22:30 [error] 7#7: *7 open() "/var/www/favicon.ico" failed (2: No such file or directory), client: 172.26.0.1, server: localhost, request: "GET /favicon.ico HTTP/1.1", host: "localhost", referrer: "http://localhost/user/login?redirect=%2F"
2020/04/09 15:22:36 [error] 7#7: *6 open() "/var/www/user/login" failed (2: No such file or directory), client: 172.26.0.1, server: localhost, request: "GET /user/login?redirect=%2F HTTP/1.1", host: "localhost"
2020/04/09 15:22:42 [error] 7#7: *6 open() "/var/www/jeecg-boot/sys/randomImage/1586445762880" failed (2: No such file or directory), client: 172.26.0.1, server: localhost, request: "GET /jeecg-boot/sys/randomImage/1586445762880?_t=1586445762 HTTP/1.1", host: "localhost", referrer: "http://localhost/user/login?redirect=%2F"
2020/04/09 15:23:13 [warn] 1#1: conflicting server name "localhost" on 0.0.0.0:80, ignored
2020/04/09 15:23:17 [error] 6#6: *1 open() "/var/www/user/login" failed (2: No such file or directory), client: 172.26.0.1, server: localhost, request: "GET /user/login?redirect=%2F HTTP/1.1", host: "localhost"
2020/04/09 15:23:18 [error] 6#6: *1 open() "/var/www/user/login" failed (2: No such file or directory), client: 172.26.0.1, server: localhost, request: "GET /user/login?redirect=%2F HTTP/1.1", host: "localhost"
2020/04/09 15:23:24 [error] 6#6: *1 open() "/var/www/jeecg-boot/sys/randomImage/1586445804725" failed (2: No such file or directory), client: 172.26.0.1, server: localhost, request: "GET /jeecg-boot/sys/randomImage/1586445804725?_t=1586445804 HTTP/1.1", host: "localhost", referrer: "http://localhost/user/login?redirect=%2F"
2020/04/09 15:23:29 [error] 6#6: *1 open() "/var/www/jeecg-boot/sys/randomImage/1586445809366" failed (2: No such file or directory), client: 172.26.0.1, server: localhost, request: "GET /jeecg-boot/sys/randomImage/1586445809366?_t=1586445809 HTTP/1.1", host: "localhost", referrer: "http://localhost/user/login?redirect=%2F"
2020/04/09 15:23:44 [error] 6#6: *1 open() "/var/www/jeecg-boot/sys/randomImage/1586445824637" failed (2: No such file or directory), client: 172.26.0.1, server: localhost, request: "GET /jeecg-boot/sys/randomImage/1586445824637?_t=1586445824 HTTP/1.1", host: "localhost", referrer: "http://localhost/user/login?redirect=%2F"
2020/04/09 15:28:04 [warn] 1#1: conflicting server name "localhost" on 0.0.0.0:80, ignored
2020/04/09 15:28:11 [error] 7#7: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.26.0.1, server: localhost, request: "GET /jeecg-boot/sys/randomImage/1586446091183?_t=1586446091 HTTP/1.1", upstream: "http://127.0.0.1:8080/jeecg-boot//sys/randomImage/1586446091183?_t=1586446091", host: "localhost", referrer: "http://localhost/user/login?redirect=%2F"
2020/04/09 15:28:14 [error] 7#7: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.26.0.1, server: localhost, request: "GET /jeecg-boot/sys/randomImage/1586446094094?_t=1586446094 HTTP/1.1", upstream: "http://127.0.0.1:8080/jeecg-boot//sys/randomImage/1586446094094?_t=1586446094", host: "localhost", referrer: "http://localhost/user/login?redirect=%2F"
2020/04/09 15:28:19 [error] 7#7: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.26.0.1, server: localhost, request: "GET /jeecg-boot/sys/randomImage/1586446099577?_t=1586446099 HTTP/1.1", upstream: "http://127.0.0.1:8080/jeecg-boot//sys/randomImage/1586446099577?_t=1586446099", host: "localhost", referrer: "http://localhost/user/login?redirect=%2F"
2020/04/09 15:29:31 [error] 7#7: *5 connect() failed (111: Connection refused) while connecting to upstream, client: 172.26.0.1, server: localhost, request: "GET /jeecg-boot/sys/randomImage/1586446170883?_t=1586446171 HTTP/1.1", upstream: "http://127.0.0.1:8080/jeecg-boot//sys/randomImage/1586446170883?_t=1586446171", host: "127.0.0.1", referrer: "http://127.0.0.1/user/login?redirect=%2F"
2020/04/09 15:29:36 [error] 7#7: *5 connect() failed (111: Connection refused) while connecting to upstream, client: 172.26.0.1, server: localhost, request: "GET /jeecg-boot/sys/randomImage/1586446175999?_t=1586446175 HTTP/1.1", upstream: "http://127.0.0.1:8080/jeecg-boot//sys/randomImage/1586446175999?_t=1586446175", host: "127.0.0.1", referrer: "http://127.0.0.1/user/login?redirect=%2F"
2020/04/09 15:29:51 [error] 7#7: *5 connect() failed (111: Connection refused) while connecting to upstream, client: 172.26.0.1, server: localhost, request: "GET /jeecg-boot/sys/randomImage/1586446191643?_t=1586446191 HTTP/1.1", upstream: "http://127.0.0.1:8080/jeecg-boot//sys/randomImage/1586446191643?_t=1586446191", host: "127.0.0.1", referrer: "http://127.0.0.1/user/login?redirect=%2F"
2020/04/09 15:30:00 [error] 7#7: *5 connect() failed (111: Connection refused) while connecting to upstream, client: 172.26.0.1, server: localhost, request: "GET /jeecg-boot/sys/randomImage/1586446200596?_t=1586446200 HTTP/1.1", upstream: "http://127.0.0.1:8080/jeecg-boot//sys/randomImage/1586446200596?_t=1586446200", host: "127.0.0.1", referrer: "http://127.0.0.1/user/login?redirect=%2F"
2020/04/09 15:30:30 [warn] 1#1: conflicting server name "localhost" on 0.0.0.0:80, ignored
2020/04/09 15:30:37 [error] 7#7: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.26.0.1, server: localhost, request: "GET /jeecg-boot/sys/randomImage/1586446237049?_t=1586446237 HTTP/1.1", upstream: "http://127.0.0.1:8080/jeecg-boot//sys/randomImage/1586446237049?_t=1586446237", host: "127.0.0.1", referrer: "http://127.0.0.1/user/login?redirect=%2F"
2020/04/09 15:30:39 [error] 7#7: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.26.0.1, server: localhost, request: "GET /jeecg-boot/sys/randomImage/1586446239753?_t=1586446239 HTTP/1.1", upstream: "http://127.0.0.1:8080/jeecg-boot//sys/randomImage/1586446239753?_t=1586446239", host: "127.0.0.1", referrer: "http://127.0.0.1/user/login?redirect=%2F"
2020/04/09 15:30:40 [error] 7#7: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.26.0.1, server: localhost, request: "GET /jeecg-boot/sys/randomImage/1586446240595?_t=1586446240 HTTP/1.1", upstream: "http://127.0.0.1:8080/jeecg-boot//sys/randomImage/1586446240595?_t=1586446240", host: "127.0.0.1", referrer: "http://127.0.0.1/user/login?redirect=%2F"
2020/04/09 15:31:23 [error] 7#7: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.26.0.1, server: localhost, request: "GET /jeecg-boot/sys/randomImage/1586446283831?_t=1586446283 HTTP/1.1", upstream: "http://127.0.0.1:8080/jeecg-boot//sys/randomImage/1586446283831?_t=1586446283", host: "127.0.0.1", referrer: "http://127.0.0.1/user/login?redirect=%2F"
2020/04/09 15:31:48 [error] 7#7: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.26.0.1, server: localhost, request: "GET /jeecg-boot/sys/randomImage/1586446308901?_t=1586446308 HTTP/1.1", upstream: "http://127.0.0.1:8080/jeecg-boot//sys/randomImage/1586446308901?_t=1586446308", host: "127.0.0.1", referrer: "http://127.0.0.1/user/login"
2020/04/09 15:31:51 [error] 7#7: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.26.0.1, server: localhost, request: "GET /jeecg-boot/sys/randomImage/1586446311460?_t=1586446311 HTTP/1.1", upstream: "http://127.0.0.1:8080/jeecg-boot//sys/randomImage/1586446311460?_t=1586446311", host: "127.0.0.1", referrer: "http://127.0.0.1/user/login"
2020/04/09 15:40:20 [error] 7#7: *8 connect() failed (111: Connection refused) while connecting to upstream, client: 172.26.0.1, server: localhost, request: "GET /jeecg-boot/sys/randomImage/1586446820468?_t=1586446820 HTTP/1.1", upstream: "http://127.0.0.1:8080/jeecg-boot//sys/randomImage/1586446820468?_t=1586446820", host: "127.0.0.1", referrer: "http://127.0.0.1/user/login?redirect=%2F"
2020/04/10 05:25:37 [warn] 1#1: conflicting server name "localhost" on 0.0.0.0:80, ignored
2020/04/10 05:55:28 [error] 8#8: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.26.0.1, server: localhost, request: "GET /jeecg-boot/sys/randomImage/1586498128209?_t=1586498128 HTTP/1.1", upstream: "http://127.0.0.1:8080/jeecg-boot//sys/randomImage/1586498128209?_t=1586498128", host: "localhost", referrer: "http://localhost/user/login?redirect=%2F"
2020/04/10 05:56:28 [error] 8#8: *6 connect() failed (111: Connection refused) while connecting to upstream, client: 172.26.0.1, server: localhost, request: "GET /jeecg-boot/sys/randomImage/1586498188085?_t=1586498188 HTTP/1.1", upstream: "http://127.0.0.1:8080/jeecg-boot//sys/randomImage/1586498188085?_t=1586498188", host: "localhost", referrer: "http://localhost/user/login?redirect=%2F"
2020/04/10 05:56:29 [error] 8#8: *6 connect() failed (111: Connection refused) while connecting to upstream, client: 172.26.0.1, server: localhost, request: "GET /jeecg-boot/sys/randomImage/1586498189686?_t=1586498189 HTTP/1.1", upstream: "http://127.0.0.1:8080/jeecg-boot//sys/randomImage/1586498189686?_t=1586498189", host: "localhost", referrer: "http://localhost/user/login?redirect=%2F"
2020/04/10 05:56:30 [error] 8#8: *6 connect() failed (111: Connection refused) while connecting to upstream, client: 172.26.0.1, server: localhost, request: "GET /jeecg-boot/sys/randomImage/1586498190648?_t=1586498190 HTTP/1.1", upstream: "http://127.0.0.1:8080/jeecg-boot//sys/randomImage/1586498190648?_t=1586498190", host: "localhost", referrer: "http://localhost/user/login?redirect=%2F"
2020/04/10 05:59:18 [error] 8#8: *10 connect() failed (111: Connection refused) while connecting to upstream, client: 172.26.0.1, server: localhost, request: "GET /jeecg-boot HTTP/1.1", upstream: "http://127.0.0.1:8080/jeecg-boot/", host: "localhost"
2020/04/10 05:59:29 [error] 8#8: *10 connect() failed (111: Connection refused) while connecting to upstream, client: 172.26.0.1, server: localhost, request: "GET /jeecg-boot/sys/randomImage/1586498369268?_t=1586498369 HTTP/1.1", upstream: "http://127.0.0.1:8080/jeecg-boot//sys/randomImage/1586498369268?_t=1586498369", host: "localhost", referrer: "http://localhost/user/login?redirect=%2F"
2020/04/10 06:04:09 [error] 8#8: *14 connect() failed (111: Connection refused) while connecting to upstream, client: 172.26.0.1, server: localhost, request: "GET /jeecg-boot/sys/randomImage/1586498648986?_t=1586498649 HTTP/1.1", upstream: "http://127.0.0.1:8080/jeecg-boot//sys/randomImage/1586498648986?_t=1586498649", host: "127.0.0.1", referrer: "http://127.0.0.1/user/login?redirect=%2F"
2020/04/10 06:04:14 [error] 8#8: *14 connect() failed (111: Connection refused) while connecting to upstream, client: 172.26.0.1, server: localhost, request: "GET /jeecg-boot/sys/randomImage/1586498654394?_t=1586498654 HTTP/1.1", upstream: "http://127.0.0.1:8080/jeecg-boot//sys/randomImage/1586498654394?_t=1586498654", host: "127.0.0.1", referrer: "http://127.0.0.1/user/login?redirect=%2F"
2020/04/10 06:11:25 [error] 8#8: *20 connect() failed (111: Connection refused) while connecting to upstream, client: 172.26.0.1, server: localhost, request: "GET /jeecg-boot/sys/randomImage/1586499084971?_t=1586499084 HTTP/1.1", upstream: "http://127.0.0.1:8080/jeecg-boot//sys/randomImage/1586499084971?_t=1586499084", host: "127.0.0.1", referrer: "http://127.0.0.1/user/login?redirect=%2F"
2020/04/10 06:14:50 [error] 8#8: *22 connect() failed (111: Connection refused) while connecting to upstream, client: 172.26.0.1, server: localhost, request: "GET /jeecg-boot/sys/randomImage/1586499290664?_t=1586499290 HTTP/1.1", upstream: "http://127.0.0.1:8080/jeecg-boot//sys/randomImage/1586499290664?_t=1586499290", host: "127.0.0.1", referrer: "http://127.0.0.1/user/login?redirect=%2F"
2020/04/10 06:15:03 [error] 8#8: *22 connect() failed (111: Connection refused) while connecting to upstream, client: 172.26.0.1, server: localhost, request: "GET /jeecg-boot/sys/user/checkOnlyUser?_t=1586499303 HTTP/1.1", upstream: "http://127.0.0.1:8080/jeecg-boot//sys/user/checkOnlyUser?_t=1586499303", host: "127.0.0.1", referrer: "http://127.0.0.1/user/register"
2020/04/10 06:15:20 [error] 8#8: *22 connect() failed (111: Connection refused) while connecting to upstream, client: 172.26.0.1, server: localhost, request: "GET /jeecg-boot/sys/user/checkOnlyUser?_t=1586499320 HTTP/1.1", upstream: "http://127.0.0.1:8080/jeecg-boot//sys/user/checkOnlyUser?_t=1586499320", host: "127.0.0.1", referrer: "http://127.0.0.1/user/register"
2020/04/10 06:16:15 [error] 8#8: *22 connect() failed (111: Connection refused) while connecting to upstream, client: 172.26.0.1, server: localhost, request: "GET /jeecg-boot/sys/randomImage/1586499374989?_t=1586499375 HTTP/1.1", upstream: "http://127.0.0.1:8080/jeecg-boot//sys/randomImage/1586499374989?_t=1586499375", host: "127.0.0.1", referrer: "http://127.0.0.1/user/login"
2020/04/10 06:16:51 [error] 8#8: *27 connect() failed (111: Connection refused) while connecting to upstream, client: 172.26.0.1, server: localhost, request: "GET /jeecg-boot/sys/randomImage/1586499411105?_t=1586499411 HTTP/1.1", upstream: "http://127.0.0.1:8080/jeecg-boot//sys/randomImage/1586499411105?_t=1586499411", host: "127.0.0.1", referrer: "http://127.0.0.1/user/login"
2020/04/10 06:16:53 [error] 8#8: *27 connect() failed (111: Connection refused) while connecting to upstream, client: 172.26.0.1, server: localhost, request: "GET /jeecg-boot/sys/randomImage/1586499413197?_t=1586499413 HTTP/1.1", upstream: "http://127.0.0.1:8080/jeecg-boot//sys/randomImage/1586499413197?_t=1586499413", host: "127.0.0.1", referrer: "http://127.0.0.1/user/login"
2020/04/10 06:19:08 [error] 8#8: *30 connect() failed (111: Connection refused) while connecting to upstream, client: 172.26.0.1, server: localhost, request: "GET /jeecg-boot/sys/randomImage/1586499548724?_t=1586499548 HTTP/1.1", upstream: "http://127.0.0.1:8080/jeecg-boot//sys/randomImage/1586499548724?_t=1586499548", host: "127.0.0.1", referrer: "http://127.0.0.1/user/login"
2020/04/10 06:19:36 [error] 8#8: *34 connect() failed (111: Connection refused) while connecting to upstream, client: 172.26.0.1, server: localhost, request: "GET /jeecg-boot/sys/randomImage/1586499576608?_t=1586499576 HTTP/1.1", upstream: "http://127.0.0.1:8080/jeecg-boot//sys/randomImage/1586499576608?_t=1586499576", host: "localhost", referrer: "http://localhost/user/login?redirect=%2F"
2020/04/10 06:19:42 [error] 8#8: *33 connect() failed (111: Connection refused) while connecting to upstream, client: 172.26.0.1, server: localhost, request: "GET /jeecg-boot/sys/randomImage/1586499582145?_t=1586499582 HTTP/1.1", upstream: "http://127.0.0.1:8080/jeecg-boot//sys/randomImage/1586499582145?_t=1586499582", host: "localhost", referrer: "http://localhost/user/login?redirect=%2F"
2020/04/10 06:19:55 [error] 8#8: *33 connect() failed (111: Connection refused) while connecting to upstream, client: 172.26.0.1, server: localhost, request: "GET /jeecg-boot/sys/randomImage/1586499595550?_t=1586499595 HTTP/1.1", upstream: "http://127.0.0.1:8080/jeecg-boot//sys/randomImage/1586499595550?_t=1586499595", host: "localhost", referrer: "http://localhost/user/login?redirect=%2F"
2020/04/10 06:20:50 [error] 8#8: *33 connect() failed (111: Connection refused) while connecting to upstream, client: 172.26.0.1, server: localhost, request: "GET /jeecg-boot/sys/randomImage/1586499650341?_t=1586499650 HTTP/1.1", upstream: "http://127.0.0.1:8080/jeecg-boot//sys/randomImage/1586499650341?_t=1586499650", host: "localhost", referrer: "http://localhost/user/login?redirect=%2F"
2020/04/10 06:20:51 [error] 8#8: *33 connect() failed (111: Connection refused) while connecting to upstream, client: 172.26.0.1, server: localhost, request: "GET /jeecg-boot/sys/randomImage/1586499651326?_t=1586499651 HTTP/1.1", upstream: "http://127.0.0.1:8080/jeecg-boot//sys/randomImage/1586499651326?_t=1586499651", host: "localhost", referrer: "http://localhost/user/login?redirect=%2F"
2020/04/10 06:21:01 [warn] 1#1: conflicting server name "localhost" on 0.0.0.0:80, ignored
2020/04/10 06:21:09 [error] 8#8: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.26.0.1, server: localhost, request: "GET /jeecg-boot/sys/randomImage/1586499669107?_t=1586499669 HTTP/1.1", upstream: "http://127.0.0.1:8080/jeecg-boot//sys/randomImage/1586499669107?_t=1586499669", host: "localhost", referrer: "http://localhost/user/login?redirect=%2F"
2020/04/10 06:21:10 [error] 8#8: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.26.0.1, server: localhost, request: "GET /jeecg-boot/sys/randomImage/1586499670896?_t=1586499670 HTTP/1.1", upstream: "http://127.0.0.1:8080/jeecg-boot//sys/randomImage/1586499670896?_t=1586499670", host: "localhost", referrer: "http://localhost/user/login?redirect=%2F"
2020/04/10 06:21:14 [error] 8#8: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.26.0.1, server: localhost, request: "GET /jeecg-boot/sys/randomImage/1586499674765?_t=1586499674 HTTP/1.1", upstream: "http://127.0.0.1:8080/jeecg-boot//sys/randomImage/1586499674765?_t=1586499674", host: "localhost", referrer: "http://localhost/user/login?redirect=%2F"
2020/04/10 06:21:16 [error] 8#8: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.26.0.1, server: localhost, request: "GET /jeecg-boot/sys/randomImage/1586499676001?_t=1586499676 HTTP/1.1", upstream: "http://127.0.0.1:8080/jeecg-boot//sys/randomImage/1586499676001?_t=1586499676", host: "localhost", referrer: "http://localhost/user/login?redirect=%2F"
2020/04/10 06:40:56 [warn] 1#1: conflicting server name "localhost" on 0.0.0.0:80, ignored
2020/04/10 06:41:07 [error] 7#7: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.26.0.1, server: localhost, request: "GET /jeecg-boot/sys/randomImage/1586500867568?_t=1586500867 HTTP/1.1", upstream: "http://127.0.0.1:8080/jeecg-boot/sys/randomImage/1586500867568?_t=1586500867", host: "127.0.0.1", referrer: "http://127.0.0.1/user/login"
2020/04/10 06:41:10 [error] 7#7: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.26.0.1, server: localhost, request: "GET /jeecg-boot/sys/randomImage/1586500870133?_t=1586500870 HTTP/1.1", upstream: "http://127.0.0.1:8080/jeecg-boot/sys/randomImage/1586500870133?_t=1586500870", host: "127.0.0.1", referrer: "http://127.0.0.1/user/login"
2020/04/10 06:41:11 [error] 7#7: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.26.0.1, server: localhost, request: "GET /jeecg-boot/sys/randomImage/1586500871278?_t=1586500871 HTTP/1.1", upstream: "http://127.0.0.1:8080/jeecg-boot/sys/randomImage/1586500871278?_t=1586500871", host: "127.0.0.1", referrer: "http://127.0.0.1/user/login"
2020/04/10 06:41:31 [error] 7#7: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.26.0.1, server: localhost, request: "GET /jeecg-boot/sys/randomImage/1586500891799?_t=1586500891 HTTP/1.1", upstream: "http://127.0.0.1:8080/jeecg-boot/sys/randomImage/1586500891799?_t=1586500891", host: "127.0.0.1", referrer: "http://127.0.0.1/user/login"
2020/04/10 06:41:34 [error] 7#7: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.26.0.1, server: localhost, request: "GET /jeecg-boot/sys/randomImage/1586500893978?_t=1586500893 HTTP/1.1", upstream: "http://127.0.0.1:8080/jeecg-boot/sys/randomImage/1586500893978?_t=1586500893", host: "127.0.0.1", referrer: "http://127.0.0.1/user/login"
2020/04/10 06:41:34 [error] 7#7: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.26.0.1, server: localhost, request: "GET /jeecg-boot/sys/randomImage/1586500894585?_t=1586500894 HTTP/1.1", upstream: "http://127.0.0.1:8080/jeecg-boot/sys/randomImage/1586500894585?_t=1586500894", host: "127.0.0.1", referrer: "http://127.0.0.1/user/login"
2020/04/10 06:41:40 [warn] 1#1: conflicting server name "localhost" on 0.0.0.0:80, ignored
2020/04/10 06:41:44 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.26.0.1, server: localhost, request: "GET /jeecg-boot/sys/randomImage/1586500904160?_t=1586500904 HTTP/1.1", upstream: "http://127.0.0.1:8080/jeecg-boot/sys/randomImage/1586500904160?_t=1586500904", host: "127.0.0.1", referrer: "http://127.0.0.1/user/login"
2020/04/10 06:41:49 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.26.0.1, server: localhost, request: "GET /jeecg-boot/sys/randomImage/1586500909658?_t=1586500909 HTTP/1.1", upstream: "http://127.0.0.1:8080/jeecg-boot/sys/randomImage/1586500909658?_t=1586500909", host: "127.0.0.1", referrer: "http://127.0.0.1/user/login"
2020/04/10 06:41:51 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.26.0.1, server: localhost, request: "GET /jeecg-boot/sys/randomImage/1586500911920?_t=1586500911 HTTP/1.1", upstream: "http://127.0.0.1:8080/jeecg-boot/sys/randomImage/1586500911920?_t=1586500911", host: "127.0.0.1", referrer: "http://127.0.0.1/user/login"
2020/04/10 06:41:53 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.26.0.1, server: localhost, request: "GET /jeecg-boot/sys/randomImage/1586500913131?_t=1586500913 HTTP/1.1", upstream: "http://127.0.0.1:8080/jeecg-boot/sys/randomImage/1586500913131?_t=1586500913", host: "127.0.0.1", referrer: "http://127.0.0.1/user/login"
2020/04/10 06:41:53 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.26.0.1, server: localhost, request: "GET /jeecg-boot/sys/randomImage/1586500913957?_t=1586500913 HTTP/1.1", upstream: "http://127.0.0.1:8080/jeecg-boot/sys/randomImage/1586500913957?_t=1586500913", host: "127.0.0.1", referrer: "http://127.0.0.1/user/login"
2020/04/10 06:41:54 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.26.0.1, server: localhost, request: "GET /jeecg-boot/sys/randomImage/1586500914663?_t=1586500914 HTTP/1.1", upstream: "http://127.0.0.1:8080/jeecg-boot/sys/randomImage/1586500914663?_t=1586500914", host: "127.0.0.1", referrer: "http://127.0.0.1/user/login"
2020/04/10 06:41:55 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.26.0.1, server: localhost, request: "GET /jeecg-boot/sys/randomImage/1586500915769?_t=1586500915 HTTP/1.1", upstream: "http://127.0.0.1:8080/jeecg-boot/sys/randomImage/1586500915769?_t=1586500915", host: "127.0.0.1", referrer: "http://127.0.0.1/user/login"
2020/04/10 06:42:18 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.26.0.1, server: localhost, request: "GET /jeecg-boot/sys/randomImage/1586500938384?_t=1586500938 HTTP/1.1", upstream: "http://127.0.0.1:8080/jeecg-boot/sys/randomImage/1586500938384?_t=1586500938", host: "127.0.0.1", referrer: "http://127.0.0.1/user/login"
2020/04/10 06:47:03 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.26.0.1, server: localhost, request: "GET /jeecg-boot/sys/randomImage/1586501223142?_t=1586501223 HTTP/1.1", upstream: "http://127.0.0.1:8080/jeecg-boot/sys/randomImage/1586501223142?_t=1586501223", host: "127.0.0.1", referrer: "http://127.0.0.1/user/login"
2020/04/10 06:47:08 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.26.0.1, server: localhost, request: "GET /jeecg-boot/sys/randomImage/1586501228697?_t=1586501228 HTTP/1.1", upstream: "http://127.0.0.1:8080/jeecg-boot/sys/randomImage/1586501228697?_t=1586501228", host: "127.0.0.1", referrer: "http://127.0.0.1/user/login"
2020/04/10 06:56:19 [error] 6#6: *2 connect() failed (111: Connection refused) while connecting to upstream, client: 172.26.0.1, server: localhost, request: "GET /jeecg-boot/sys/randomImage/1586501779431?_t=1586501779 HTTP/1.1", upstream: "http://127.0.0.1:8080/jeecg-boot/sys/randomImage/1586501779431?_t=1586501779", host: "127.0.0.1", referrer: "http://127.0.0.1/user/login?redirect=%2F"
2020/04/10 06:56:22 [error] 6#6: *2 connect() failed (111: Connection refused) while connecting to upstream, client: 172.26.0.1, server: localhost, request: "GET /jeecg-boot/sys/randomImage/1586501782504?_t=1586501782 HTTP/1.1", upstream: "http://127.0.0.1:8080/jeecg-boot/sys/randomImage/1586501782504?_t=1586501782", host: "127.0.0.1", referrer: "http://127.0.0.1/user/login?redirect=%2F"
2020/04/10 06:56:24 [error] 6#6: *2 connect() failed (111: Connection refused) while connecting to upstream, client: 172.26.0.1, server: localhost, request: "GET /jeecg-boot/sys/randomImage/1586501784049?_t=1586501784 HTTP/1.1", upstream: "http://127.0.0.1:8080/jeecg-boot/sys/randomImage/1586501784049?_t=1586501784", host: "127.0.0.1", referrer: "http://127.0.0.1/user/login?redirect=%2F"
2020/04/10 06:56:49 [error] 6#6: *2 connect() failed (111: Connection refused) while connecting to upstream, client: 172.26.0.1, server: localhost, request: "GET /jeecg-boot/sys/randomImage/1586501809518?_t=1586501809 HTTP/1.1", upstream: "http://127.0.0.1:8080/jeecg-boot/sys/randomImage/1586501809518?_t=1586501809", host: "127.0.0.1", referrer: "http://127.0.0.1/user/login?redirect=%2F"
2020/04/10 06:57:02 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.26.0.1, server: localhost, request: "GET /jeecg-boot/sys/randomImage/1586501821906?_t=1586501822 HTTP/1.1", upstream: "http://127.0.0.1:8080/jeecg-boot/sys/randomImage/1586501821906?_t=1586501822", host: "localhost", referrer: "http://localhost/user/login?redirect=%2F"
2020/04/10 06:57:03 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.26.0.1, server: localhost, request: "GET /jeecg-boot/sys/randomImage/1586501823479?_t=1586501823 HTTP/1.1", upstream: "http://127.0.0.1:8080/jeecg-boot/sys/randomImage/1586501823479?_t=1586501823", host: "localhost", referrer: "http://localhost/user/login?redirect=%2F"
2020/04/10 06:57:04 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.26.0.1, server: localhost, request: "GET /jeecg-boot/sys/randomImage/1586501824364?_t=1586501824 HTTP/1.1", upstream: "http://127.0.0.1:8080/jeecg-boot/sys/randomImage/1586501824364?_t=1586501824", host: "localhost", referrer: "http://localhost/user/login?redirect=%2F"
2020/04/10 07:00:36 [emerg] 1#1: "server" directive is not allowed here in /etc/nginx/nginx.conf:1
2020/04/10 07:00:36 [emerg] 1#1: "server" directive is not allowed here in /etc/nginx/nginx.conf:1
2020/04/10 07:00:37 [emerg] 1#1: "server" directive is not allowed here in /etc/nginx/nginx.conf:1
2020/04/10 07:00:38 [emerg] 1#1: "server" directive is not allowed here in /etc/nginx/nginx.conf:1
2020/04/10 07:00:39 [emerg] 1#1: "server" directive is not allowed here in /etc/nginx/nginx.conf:1
2020/04/10 07:00:41 [emerg] 1#1: "server" directive is not allowed here in /etc/nginx/nginx.conf:1
2020/04/10 07:00:45 [emerg] 1#1: "server" directive is not allowed here in /etc/nginx/nginx.conf:1
2020/04/10 07:00:51 [emerg] 1#1: "server" directive is not allowed here in /etc/nginx/nginx.conf:1
2020/04/10 07:01:04 [emerg] 1#1: "server" directive is not allowed here in /etc/nginx/nginx.conf:1
2020/04/10 07:01:30 [emerg] 1#1: open() "/etc/nginx/nginx.conf" failed (2: No such file or directory)
2020/04/10 07:01:30 [emerg] 1#1: open() "/etc/nginx/nginx.conf" failed (2: No such file or directory)
2020/04/10 07:01:31 [emerg] 1#1: open() "/etc/nginx/nginx.conf" failed (2: No such file or directory)
2020/04/10 07:01:32 [emerg] 1#1: open() "/etc/nginx/nginx.conf" failed (2: No such file or directory)
2020/04/10 07:01:33 [emerg] 1#1: open() "/etc/nginx/nginx.conf" failed (2: No such file or directory)
2020/04/10 07:01:35 [emerg] 1#1: open() "/etc/nginx/nginx.conf" failed (2: No such file or directory)
2020/04/10 07:01:39 [emerg] 1#1: open() "/etc/nginx/nginx.conf" failed (2: No such file or directory)
2020/04/10 15:10:22 [error] 8#8: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.26.0.1, server: localhost, request: "GET /jeecg-boot/sys/randomImage/1586502622029?_t=1586502622 HTTP/1.1", upstream: "http://127.0.0.1:8080/jeecg-boot/sys/randomImage/1586502622029?_t=1586502622", host: "localhost", referrer: "http://localhost/user/login?redirect=%2F"
2020/04/10 15:10:24 [error] 8#8: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.26.0.1, server: localhost, request: "GET /jeecg-boot/sys/randomImage/1586502624639?_t=1586502624 HTTP/1.1", upstream: "http://127.0.0.1:8080/jeecg-boot/sys/randomImage/1586502624639?_t=1586502624", host: "localhost", referrer: "http://localhost/user/login?redirect=%2F"
2020/04/10 15:11:24 [error] 8#8: *4 connect() failed (111: Connection refused) while connecting to upstream, client: 172.26.0.1, server: localhost, request: "GET /jeecg-boot/sys/randomImage/1586502684732?_t=1586502684 HTTP/1.1", upstream: "http://127.0.0.1:8080/jeecg-boot/sys/randomImage/1586502684732?_t=1586502684", host: "localhost", referrer: "http://localhost/user/login?redirect=%2F"
2020/04/10 15:11:28 [error] 8#8: *4 connect() failed (111: Connection refused) while connecting to upstream, client: 172.26.0.1, server: localhost, request: "GET /jeecg-boot/sys/randomImage/1586502688215?_t=1586502688 HTTP/1.1", upstream: "http://127.0.0.1:8080/jeecg-boot/sys/randomImage/1586502688215?_t=1586502688", host: "localhost", referrer: "http://localhost/user/login?redirect=%2F"
2020/04/10 15:11:29 [error] 8#8: *4 connect() failed (111: Connection refused) while connecting to upstream, client: 172.26.0.1, server: localhost, request: "GET /jeecg-boot/sys/randomImage/1586502688989?_t=1586502688 HTTP/1.1", upstream: "http://127.0.0.1:8080/jeecg-boot/sys/randomImage/1586502688989?_t=1586502688", host: "localhost", referrer: "http://localhost/user/login?redirect=%2F"
2020/04/10 15:15:23 [error] 7#7: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.26.0.1, server: localhost, request: "GET /jeecg-boot/sys/randomImage/1586502923039?_t=1586502923 HTTP/1.1", upstream: "http://127.0.0.1:8080/jeecg-boot//sys/randomImage/1586502923039?_t=1586502923", host: "localhost", referrer: "http://localhost/user/login?redirect=%2F"
2020/04/10 15:15:26 [error] 7#7: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.26.0.1, server: localhost, request: "GET /jeecg-boot/sys/randomImage/1586502926770?_t=1586502926 HTTP/1.1", upstream: "http://127.0.0.1:8080/jeecg-boot//sys/randomImage/1586502926770?_t=1586502926", host: "localhost", referrer: "http://localhost/user/login?redirect=%2F"
2020/04/10 15:15:27 [error] 7#7: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.26.0.1, server: localhost, request: "GET /jeecg-boot/sys/randomImage/1586502927921?_t=1586502927 HTTP/1.1", upstream: "http://127.0.0.1:8080/jeecg-boot//sys/randomImage/1586502927921?_t=1586502927", host: "localhost", referrer: "http://localhost/user/login?redirect=%2F"
2020/04/10 15:15:28 [error] 7#7: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.26.0.1, server: localhost, request: "GET /jeecg-boot/sys/randomImage/1586502928651?_t=1586502928 HTTP/1.1", upstream: "http://127.0.0.1:8080/jeecg-boot//sys/randomImage/1586502928651?_t=1586502928", host: "localhost", referrer: "http://localhost/user/login?redirect=%2F"
2020/04/10 15:19:36 [emerg] 1#1: open() "/etc/nginx/nginx.conf" failed (2: No such file or directory)
2020/04/10 15:19:37 [emerg] 1#1: open() "/etc/nginx/nginx.conf" failed (2: No such file or directory)
2020/04/10 15:19:38 [emerg] 1#1: open() "/etc/nginx/nginx.conf" failed (2: No such file or directory)
2020/04/10 15:19:38 [emerg] 1#1: open() "/etc/nginx/nginx.conf" failed (2: No such file or directory)
2020/04/10 15:19:40 [emerg] 1#1: open() "/etc/nginx/nginx.conf" failed (2: No such file or directory)
2020/04/10 15:19:42 [emerg] 1#1: open() "/etc/nginx/nginx.conf" failed (2: No such file or directory)
2020/04/10 15:19:45 [emerg] 1#1: open() "/etc/nginx/nginx.conf" failed (2: No such file or directory)
2020/04/10 15:19:52 [emerg] 1#1: open() "/etc/nginx/nginx.conf" failed (2: No such file or directory)
2020/04/10 15:20:05 [emerg] 1#1: open() "/etc/nginx/nginx.conf" failed (2: No such file or directory)
2020/04/10 15:20:31 [emerg] 1#1: open() "/etc/nginx/nginx.conf" failed (2: No such file or directory)
2020/04/10 15:21:23 [emerg] 1#1: open() "/etc/nginx/nginx.conf" failed (2: No such file or directory)
2020/04/10 15:22:09 [emerg] 1#1: open() "/etc/nginx/nginx.conf" failed (2: No such file or directory)
2020/04/10 15:22:09 [emerg] 1#1: open() "/etc/nginx/nginx.conf" failed (2: No such file or directory)
2020/04/10 15:22:10 [emerg] 1#1: open() "/etc/nginx/nginx.conf" failed (2: No such file or directory)
2020/04/10 15:22:11 [emerg] 1#1: open() "/etc/nginx/nginx.conf" failed (2: No such file or directory)
2020/04/10 15:22:12 [emerg] 1#1: open() "/etc/nginx/nginx.conf" failed (2: No such file or directory)
2020/04/10 15:22:14 [emerg] 1#1: open() "/etc/nginx/nginx.conf" failed (2: No such file or directory)
2020/04/10 15:22:18 [emerg] 1#1: open() "/etc/nginx/nginx.conf" failed (2: No such file or directory)
2020/04/10 15:22:24 [emerg] 1#1: "server" directive is not allowed here in /etc/nginx/nginx.conf:1
2020/04/10 15:22:38 [emerg] 1#1: "server" directive is not allowed here in /etc/nginx/nginx.conf:1
2020/04/10 15:22:46 [emerg] 1#1: "server" directive is not allowed here in /etc/nginx/nginx.conf:1
2020/04/10 15:22:46 [emerg] 1#1: "server" directive is not allowed here in /etc/nginx/nginx.conf:1
2020/04/10 15:22:47 [emerg] 1#1: "server" directive is not allowed here in /etc/nginx/nginx.conf:1
2020/04/10 15:22:48 [emerg] 1#1: "server" directive is not allowed here in /etc/nginx/nginx.conf:1
2020/04/10 15:22:49 [emerg] 1#1: "server" directive is not allowed here in /etc/nginx/nginx.conf:1
2020/04/10 15:22:51 [emerg] 1#1: "server" directive is not allowed here in /etc/nginx/nginx.conf:1
2020/04/10 15:22:55 [emerg] 1#1: "server" directive is not allowed here in /etc/nginx/nginx.conf:1
2020/04/10 15:23:02 [emerg] 1#1: "server" directive is not allowed here in /etc/nginx/nginx.conf:1
2020/04/10 15:23:15 [emerg] 1#1: "server" directive is not allowed here in /etc/nginx/nginx.conf:1
2020/04/10 15:23:41 [emerg] 1#1: "server" directive is not allowed here in /etc/nginx/nginx.conf:1
2020/04/10 15:24:19 [error] 7#7: *1 open() "/usr/share/nginx/html/user/login" failed (2: No such file or directory), client: 172.26.0.1, server: localhost, request: "GET /user/login?redirect=%2F HTTP/1.1", host: "127.0.0.1"
2020/04/10 15:24:19 [error] 7#7: *2 open() "/usr/share/nginx/html/favicon.ico" failed (2: No such file or directory), client: 172.26.0.1, server: localhost, request: "GET /favicon.ico HTTP/1.1", host: "127.0.0.1", referrer: "http://127.0.0.1/user/login?redirect=%2F"
2020/04/10 15:26:11 [error] 6#6: *2 open() "/usr/share/nginx/html/css/chunk-vendors.8c439b52.css" failed (2: No such file or directory), client: 172.26.0.1, server: localhost, request: "GET /css/chunk-vendors.8c439b52.css HTTP/1.1", host: "localhost", referrer: "http://localhost/"
2020/04/10 15:26:11 [error] 6#6: *1 open() "/usr/share/nginx/html/js/app.f396cfe6.js" failed (2: No such file or directory), client: 172.26.0.1, server: localhost, request: "GET /js/app.f396cfe6.js HTTP/1.1", host: "localhost", referrer: "http://localhost/"
2020/04/10 15:26:11 [error] 6#6: *3 open() "/usr/share/nginx/html/css/app.92bb2644.css" failed (2: No such file or directory), client: 172.26.0.1, server: localhost, request: "GET /css/app.92bb2644.css HTTP/1.1", host: "localhost", referrer: "http://localhost/"
2020/04/10 15:26:11 [error] 6#6: *4 open() "/usr/share/nginx/html/js/chunk-vendors.755c8283.js" failed (2: No such file or directory), client: 172.26.0.1, server: localhost, request: "GET /js/chunk-vendors.755c8283.js HTTP/1.1", host: "localhost", referrer: "http://localhost/"
2020/04/10 15:28:29 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.26.0.1, server: localhost, request: "GET /jeecg-boot/sys/randomImage/1586503709101?_t=1586503709 HTTP/1.1", upstream: "http://127.0.0.1:8080/jeecg-boot//sys/randomImage/1586503709101?_t=1586503709", host: "localhost", referrer: "http://localhost/user/login?redirect=%2F"
2020/04/10 15:28:32 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.26.0.1, server: localhost, request: "GET /jeecg-boot/sys/randomImage/1586503712508?_t=1586503712 HTTP/1.1", upstream: "http://127.0.0.1:8080/jeecg-boot//sys/randomImage/1586503712508?_t=1586503712", host: "localhost", referrer: "http://localhost/user/login?redirect=%2F"
2020/04/10 15:28:33 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.26.0.1, server: localhost, request: "GET /jeecg-boot/sys/randomImage/1586503713868?_t=1586503713 HTTP/1.1", upstream: "http://127.0.0.1:8080/jeecg-boot//sys/randomImage/1586503713868?_t=1586503713", host: "localhost", referrer: "http://localhost/user/login?redirect=%2F"
2020/04/10 15:28:34 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.26.0.1, server: localhost, request: "GET /jeecg-boot/sys/randomImage/1586503714499?_t=1586503714 HTTP/1.1", upstream: "http://127.0.0.1:8080/jeecg-boot//sys/randomImage/1586503714499?_t=1586503714", host: "localhost", referrer: "http://localhost/user/login?redirect=%2F"
2020/04/10 15:28:48 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.26.0.1, server: localhost, request: "GET /jeecg-boot/sys/randomImage/1586503728568?_t=1586503728 HTTP/1.1", upstream: "http://127.0.0.1:8080/jeecg-boot//sys/randomImage/1586503728568?_t=1586503728", host: "localhost", referrer: "http://localhost/user/login?redirect=%2F"
2020/04/10 15:30:34 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.26.0.1, server: localhost, request: "GET /jeecg-boot/sys/randomImage/1586503834418?_t=1586503834 HTTP/1.1", upstream: "http://127.0.0.1:8080/jeecg-boot//sys/randomImage/1586503834418?_t=1586503834", host: "localhost", referrer: "http://localhost/user/login?redirect=%2F"
2020/04/10 15:30:36 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.26.0.1, server: localhost, request: "GET /jeecg-boot/sys/randomImage/1586503836599?_t=1586503836 HTTP/1.1", upstream: "http://127.0.0.1:8080/jeecg-boot//sys/randomImage/1586503836599?_t=1586503836", host: "localhost", referrer: "http://localhost/user/login?redirect=%2F"
2020/04/10 15:30:37 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.26.0.1, server: localhost, request: "GET /jeecg-boot/sys/randomImage/1586503837332?_t=1586503837 HTTP/1.1", upstream: "http://127.0.0.1:8080/jeecg-boot//sys/randomImage/1586503837332?_t=1586503837", host: "localhost", referrer: "http://localhost/user/login?redirect=%2F"
2020/04/10 15:30:37 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.26.0.1, server: localhost, request: "GET /jeecg-boot/sys/randomImage/1586503837818?_t=1586503837 HTTP/1.1", upstream: "http://127.0.0.1:8080/jeecg-boot//sys/randomImage/1586503837818?_t=1586503837", host: "localhost", referrer: "http://localhost/user/login?redirect=%2F"
2020/04/10 15:40:54 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.26.0.1, server: localhost, request: "GET /jeecg-boot/sys/randomImage/1586504454726?_t=1586504454 HTTP/1.1", upstream: "http://127.0.0.1:8080/jeecg-boot//sys/randomImage/1586504454726?_t=1586504454", host: "localhost", referrer: "http://localhost/user/login?redirect=%2F"
2020/04/10 15:40:57 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.26.0.1, server: localhost, request: "GET /jeecg-boot/sys/randomImage/1586504457738?_t=1586504457 HTTP/1.1", upstream: "http://127.0.0.1:8080/jeecg-boot//sys/randomImage/1586504457738?_t=1586504457", host: "localhost", referrer: "http://localhost/user/login?redirect=%2F"
2020/04/10 15:40:58 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.26.0.1, server: localhost, request: "GET /jeecg-boot/sys/randomImage/1586504458939?_t=1586504458 HTTP/1.1", upstream: "http://127.0.0.1:8080/jeecg-boot//sys/randomImage/1586504458939?_t=1586504458", host: "localhost", referrer: "http://localhost/user/login?redirect=%2F"
2020/04/10 15:45:02 [error] 7#7: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.26.0.1, server: localhost, request: "GET /jeecg-boot/sys/randomImage/1586504702919?_t=1586504702 HTTP/1.1", upstream: "http://127.0.0.1:8080/jeecg-boot//sys/randomImage/1586504702919?_t=1586504702", host: "localhost", referrer: "http://localhost/user/login?redirect=%2F"
2020/04/10 15:45:06 [error] 7#7: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.26.0.1, server: localhost, request: "GET /jeecg-boot/sys/randomImage/1586504706251?_t=1586504706 HTTP/1.1", upstream: "http://127.0.0.1:8080/jeecg-boot//sys/randomImage/1586504706251?_t=1586504706", host: "localhost", referrer: "http://localhost/user/login?redirect=%2F"
2020/04/10 15:45:07 [error] 7#7: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.26.0.1, server: localhost, request: "GET /jeecg-boot/sys/randomImage/1586504707543?_t=1586504707 HTTP/1.1", upstream: "http://127.0.0.1:8080/jeecg-boot//sys/randomImage/1586504707543?_t=1586504707", host: "localhost", referrer: "http://localhost/user/login?redirect=%2F"
2020/04/10 15:45:08 [error] 7#7: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.26.0.1, server: localhost, request: "GET /jeecg-boot/sys/randomImage/1586504708445?_t=1586504708 HTTP/1.1", upstream: "http://127.0.0.1:8080/jeecg-boot//sys/randomImage/1586504708445?_t=1586504708", host: "localhost", referrer: "http://localhost/user/login?redirect=%2F"
